data/mods/Xedra_Evolved/monsters/changeling.json

```diff
@@ -538,7 +538,7 @@
     "harvest": "demihuman",
     "weakpoint_sets": [ "wps_humanoid_body" ],
     "families": [ "prof_wp_demihuman" ],
-    "special_attacks": [ { "id": "ranged_pull", "cooldown": 10 }, [ "GRAB_DRAG", 3 ] ],
+    "special_attacks": [ { "id": "ranged_pull", "cooldown": 10 }, { "id": "grab_drag" }, { "id": "drag_followup" } ],
```

Comment:
So what I currently have setup will at least sometimes drag it's victims into deep water where they start drowning.   Will this work the same way?

Reply:
It will drag them once, then on the next attack drag them again, repeat every two turns as long as the grab is maintained (and they get ten turns cooldown on the first grab). If they have a way to deal with water's movecost they will be pretty consistent about it.

Reply:
You rock!